### PR TITLE
updating storage cost queries to correct storage and total values

### PR DESCRIPTION
**Pull request by easterbrennan**
- Updated cost by cluster dashboard description text.
- Updated the storage cost query, the change was:
`sum(container_fs_limit_bytes{device=~"^/dev/[sv]d[a-z][1-9]$",id!="/"}) / 1024 / 1024 / 1024 * $costStorageSSD`
to
`sum(node_filesystem_size_bytes{mountpoint="/mnt/stateful_partition"} + $unaccountedNodeStorage) / 1024 / 1024 / 1024 * $costStorageSSD`
- unaccounted node storage has been added as a variable

### CODE REVIEW

File: lib/mintel/dashboards/components/panels/cost-analysis.libsonnet [hunk 1]
```diff
@@ -89,7 +89,7 @@ local seriesOverrides = import 'components/series_overrides.libsonnet';
           by (persistentvolumeclaim, namespace, storageclass) + on (persistentvolumeclaim, namespace)
           group_right(storageclass) sum(kube_persistentvolumeclaim_resource_requests_storage_bytes)
           by (persistentvolumeclaim, namespace)) / 1024 / 1024 / 1024 or on() vector(0) )* $costStorageStandard
-          + sum(container_fs_limit_bytes{device=~"^/dev/[sv]d[a-z][1-9]$",id!="/"}) / 1024 / 1024 / 1024 * $costStorageSSD
+          + sum(node_filesystem_size_bytes{mountpoint="/mnt/stateful_partition"} + $unaccountedNodeStorage) / 1024 / 1024 / 1024 * $costStorageSSD
```
Comment (nabadger) [inline lib/mintel/dashboards/components/panels/cost-analysis.libsonnet]: We shouldn't be hardcoding any mount points - all available in the config

See `hostMountpointSelector`, or potentially `fstypeSelector` in `config.libsonnet`

That way, when GKE come along and change partitions, we just have to change it in the config.

There's a few examples where this is used already.

Comment (easterbrennan) [reply lib/mintel/dashboards/components/panels/cost-analysis.libsonnet]: Ok that makes sense, I have changed it to use a var from config

File: lib/mintel/config.libsonnet [hunk 2]
```diff
@@ -79,6 +82,7 @@
     cost_storage_standard: '0.040',
     cost_ram: '2.38',
     cost_pram: '0.71',
+    unaccounted_node_storage: '4000000000',
```
Comment (primeroz) [inline lib/mintel/config.libsonnet]: can we add a comment here to explain what this is ?

Comment (primeroz) [reply lib/mintel/config.libsonnet]: and add the bytes suffix so we know the unit

**Pull request merged**